### DISCUSSION TITLE
github copilot is not working

### DISCUSSION THREAD

**Discussion by Gabriel18017** [community/community#8442] [Copilot Conversations]
sorry for my english, my dudes, i was selected in the waitlist but the copilot give me this error in the vs:
[INFO] [auth] [2021-12-06T02:39:33.549Z] Invalid copilot token: invalid expires_at:1638697201 now:1638758373
[ERROR] [default] [2021-12-06T02:39:33.564Z] GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"

**Comment by davecheney** [comment]
It looks like the clock on your computer may be wrong.

**Reply by Gabriel18017** [reply]
my clock is fine

**Reply by N3RDIUM** [reply]
Thanks! It helped a lot. ![image](https://user-images.githubusercontent.com/74598401/145070560-0cdf5d56-d15c-4de2-a4b4-ebd4b61bdaf4.png)
